CMakeLists.txt

```diff
@@ -977,7 +992,7 @@ main(int argc, char **argv)
         IF ("${IMPLEMENTATION}" MATCHES "^OPENSSL$" AND OPENSSL_FOUND)
           INCLUDE_DIRECTORIES(${OPENSSL_INCLUDE_DIR})
 	  LIST(APPEND ADDITIONAL_LIBS ${OPENSSL_LIBRARIES})
-	  LIST(REMOVE_DUPLICATES ADDITIONAL_LIBS)
+	  #LIST(REMOVE_DUPLICATES ADDITIONAL_LIBS)
```

Comment:
Do we want to get rid of this?

Reply:
`REMOVE_DUPLICATES` has the potential to break static linking with "traditional linkers". Independent libs must come at the end of the list, but that command leaves only the first occurence.
In addition, it will break multi-config keywords (`optimized`, `debug`).